lib/opencensus/stats/aggregation_data/count.rb

```diff
@@ -24,7 +24,7 @@ def initialize
         # Increment counter.
         # @param [Value] value
         # @param [Time] time Time of data point was recorded
-        def add value, time
+        def add value, time, attachments: nil
```

Comment:
Document the type for attachments. (Even if it is unused, document that fact.)

Reply:
Added doc.

Reply:
Can you document that it is unused?